src/OrchardCore/OrchardCore.Users.Abstractions/Services/IExternalLoginUserToRelateFinder.cs

```diff
@@ -0,0 +1,14 @@
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace OrchardCore.Users.Abstractions;
+
+public interface IExternalLoginUserToRelateFinder
```

Comment:
Not sure if this is a good name. Maybe `IExternalLoginUserProvider`?

Reply:
sure. Also please add comments for this interface. Explain what it is meant for. Also add a comment on every method to explain what they are intended for.

Reply:
fixed in 4726652 ... hope comments goes well